src/activities/content/ContentWebLinkEntity.js

```diff
@@ -138,6 +138,51 @@ export class ContentWebLinkEntity extends Entity {
 		this.dispose();
 	}
 
+	/**
+	 * performs a checkout action on the web link
+	 */
+	async checkoutWebLink() {
+		if (!this._entity) {
+			return;
+		}
+		const action = this._entity.getActionByName(Actions.webLink.checkoutWebLink);
+		if (!action) {
+			return;
+		}
+
+		return await performSirenAction(this._token, action);
+	}
+
+	/**
+	 * performs a cancel action on the web link
+	 */
+	async cancelWebLink() {
```

Comment:
Maybe I'm being overly explicit here, but IMO maybe these function names should have `working copy` somewhere in there. I was reading the `activities` pr and it can be _slightly_ confusing that the `cancel` refers to working copy specifically.

Reply:
I see what you're saying, that's probably a good call. Should we do that in this PR (and the associated Activities PR) or as a follow-up?

Reply:
Followup seems fine, no reason to hold this up

Reply:
@ianlaidlaw , now that the `cancel` action no longer exists, do you still feel as strongly about adding `workingCopy` to the others? I feel like `checkout` and `commit` are a lot more explicit than cancel was and it is maybe less necessary now, but interested in your thoughts.

Reply:
seems fine to me!